draft-ietf-quic-transport.md

```diff
@@ -2538,10 +2538,10 @@ ACK Block (repeated):
 Implementations MUST NOT generate packets that only contain ACK frames in
 response to packets which only contain ACK frames. However, they MUST
 acknowledge packets containing only ACK frames when sending ACK frames in
-response to other packets.  Implementations MUST NOT send more than one ACK
-frame per received packet that contains frames other than ACK frames.  Packets
-containing non-ACK frames MUST be acknowledged immediately or when a delayed
-ack timer expires.
+response to other packets.  Implementations MUST NOT send more than one packet
+containing only ACK frames per received packet that contains frames other than
```

Comment:
I have no objection, but is the motivation for this the coming CRYPTO_ACK/ACK split?

Reply:
No. It was purely that the text didn't match my expectations